pkg/chartutil/create.go

```diff
@@ -147,16 +147,13 @@ service:
 
 ingress:
   enabled: false
+  # className: nginx
   annotations: {}
-    # kubernetes.io/ingress.class: nginx
     # kubernetes.io/tls-acme: "true"
-  hosts:
+  rules:
```

Comment:
This is also a significant change in the template, but I actually like it.

Reply:
Breaking changes to `helm create` cannot be made as per the backwards compatibility policy. Please revert.

Reply:
I am aware of the change... it's an alignment to ingress v1

helm create is a template for a chart. I do not see why there is a compatibility needed.
once created, the chart will even if you delete the create command at whole. also there is no "upgrade" functionallity.

as I wrote this change is for asking the community. I'm totally fine to close this pull request and ignore the change.

Reply:
> I do not see why there is a compatibility needed.

It's needed because people tend to follow the helm template and as a result a lot of charts look the same, making it easy for people to start using new charts. It's also very easy to overlook a change like this leading to endless hours of debugging.